docs/docs/main/about/background/nemo2.md

```diff
@@ -102,24 +104,24 @@ kinds of parallelism if a micro-batch fits on a single device still holds. Split
 represent the fewest necessary communications between GPUs on your cluster, so standard DDP should run the fastest if
 you can get your training loop for a micro batch to fit on one GPU.
 
-#### Mixing parallelism strategies
+#### Mixing Parallelism Strategies
 
-You can mix multiple kinds of parallelism together to achieve a more performant result. In general experimentation
+You can mix different kinds of parallelism together to achieve a better result. In general, experimentation
 should be done to identify the optimal mix of parallelism. See this
 [YouTube tutorial from Jared Casper](https://youtu.be/gHaNUcS1_O4) for more background on megatron parallelism
 strategies.
 
-Below is a figure demonstrating how mixing strategies results in larger "virtual GPUs", which similarly means you have
-fewer distinct micro-batches in flight across your cluster. Also note that the number of virtual GPUs is multiplicative
-so if you have `TP=2` and `PP=2` then you are creating a larger virtual GPU out of `2*2=4` GPUs, so your cluster size
+The figure below demonstrates how mixing strategies results in larger "virtual GPUs", which similarly means you have
+fewer distinct micro-batches in flight across your cluster. Note that the number of virtual GPUs is multiplicative
+so if you have `TP=2` and `PP=2`, then you are creating a larger virtual GPU out of `2*2=4` GPUs, so your cluster size
 needs to be a multiple of 4 in this case.
 ![Mixing Tensor and Pipeline Parallelism](../../../assets/images/megatron_background/tensor_and_pipeline_parallelism.png)
 
-#### Scheduling model parallelism
+#### Scheduling Model Parallelism
 
-You can improve on naive schedules by splitting up micro-batches into smaller pieces, executing multiple stages of the
+You can improve on naive schedules by splitting up micro-batches into smaller pieces, executing many stages of the
 model on single GPUs, and starting computing the backwards pass of one micro-batch while another is going through forward.
-These optimizations allow for better cluster GPU utilization to be achieved. For example the following figure shows
-how more advanced splitting techniques in megatron (eg the interleaved scheduler) provide better utilization when model
-parallelism is used. Again when you can get away without using model parallelism (DDP), that is generally the best approach.
+These optimizations allow for better cluster GPU utilization to be achieved. For example, the following figure shows
+how more advanced splitting techniques in megatron (for example, the interleaved scheduler) offer better utilization when model
+parallelism is used. As best possible, we don't recommend using model parallelism (DDP).
```

Comment:
to be honest I don't understand this sentence, particularly the adverbial phrase "As best possible" used here:

> As best possible, we don't recommend using model parallelism (DDP)

I think it's missing an "as", but I still don't understand that phrasing: "As best as possible, we don't recommend using model parallelism (DDP)". The first clause still confuses me.